docs/workshop/onboarding.md

```diff
@@ -1,3 +1,74 @@
+
+In this workshop, you will learn how to collect data from an open CI/CD data source, use Jupyterlab and its extensions to manage dependencies, and run jupyter notebooks. You will learn how to perform automated, repeatable experiments using Elyra and Kubeflow Pipelines, create and deploy machine learning modes on OpenShift using Seldon and visualize results on a Superset dashboard.
+
+To get started with this tour, let’s first introduce you to the various tools that are available on Operate First environment and will help us in building the AIOps application using Operate First.
+
+
+
+1. **Openshift Console -**
+
+The Operate First initiative currently manages two clusters within the [MOC environment](https://massopen.cloud/).
+
+* [Smaug cluster](https://console-openshift-console.apps.smaug.na.operate-first.cloud/) for all user workloads
+* [Infra cluster](https://console-openshift-console.apps.moc-infra.massopen.cloud/) for cluster management, housing ACM and ArgoCD
+
+For this workshop, we shall proceed with using the [Smaug cluster](https://console-openshift-console.apps.smaug.na.operate-first.cloud/).
+
+![Cluster Login Page](../assets/images/cluster1.png)
+
+Select login with `operate-first`, and login into the cluster by using your Github credentials.
+
+![Cluster Landing Page](../assets/images/cluster2.png)
+
+2. **Ceph S3 storage -**
+
+Ceph provides a unified storage service with an object, block, and file interface from a single cluster built from commodity hardware components. Operate First hosts a public ceph storage bucket that is accessible by anyone. You can find the credentials and other information for this bucket below:
+
+
+```
+Endpoint:  https://s3-openshift-storage.apps.smaug.na.operate-first.cloud
+
+Bucket Name: opf-datacatalog
+
+Access Key: INDuwCuYntikCsRQioSk
+
+Secret Key: YjLeEDP+Lj30BJTbzgy84b3pzXPpWtlYnRcMYW2/
+
+Total Size: 15G
+```
+
+
+**_Some things to note:_**
+
+
+
+* Do NOT store any confidential information on this bucket as it is open to the public
+* Always back up all your data regularly, as anyone can go in and wipe it
+3. **Jupyterhub -**
+
+JupyterHub is the best way to serve Jupyter notebook for multiple users. It is a multi-user Hub that spawns, manages, and proxies multiple instances of the single-user Jupyter notebook server. Access to operate first JupyterHub is open to the public. All you need is a GitHub account.
+
+* Our JupyterHub instance on the smaug cluster can be found [here](https://jupyterhub-opf-jupyterhub.apps.smaug.na.operate-first.cloud/).
+* Login using the operate-first option, and you will be redirected to the JH spawner UI.
+
+![JH Landing Page](../assets/images/JH_landing_page.png)
+
+* You can select any notebook image that you want to try on, for this workshop, we will be working with the `ocp-ci-analysis` image.
```

Comment:
For starters, it might be confusing since `ocp-ci-analysis` name is not in the list in the pic. 

```suggestion
* You can select any notebook image that you want to try on, for this workshop, we will be working with the `Openshift CI Analysis Notebook Image` image.
```